src/org/zaproxy/zap/extension/dynssl/ExtensionDynSSL.java

```diff
@@ -204,4 +323,74 @@ private void warnRooCaCertExpired() {
 		}
 		logger.warn(warnMsg);
 	}
+
+	@Override
+	public void execute(CommandLineArgument[] args) {
+		if (arguments[ARG_CERT_LOAD].isEnabled()) {
+			File file = new File(arguments[ARG_CERT_LOAD].getArguments().firstElement());
+			if (!file.canRead()) {
+				CommandLine.error(Constant.messages.getString("dynssl.cmdline.error.noread", file.getAbsolutePath()));
+			} else {
+				String error = importRootCaCertificate(file);
+				if (error == null) {
+					CommandLine
+							.info(Constant.messages.getString("dynssl.cmdline.certload.done", file.getAbsolutePath()));
+				} else {
+					CommandLine.error(error);
+				}
+			}
+		}
+		if (arguments[ARG_CERT_PUB_DUMP].isEnabled()) {
+			File file = new File(arguments[ARG_CERT_PUB_DUMP].getArguments().firstElement());

```

Comment:
Since the dump logic is the same for pub/full it could be extracted a method `private static void writeCert(String path, CertWriter writer)` with `CertWriter` as:
 `private interface CertWriter { void write(File file) throws Exception; }`
and then just call `writeCert(arguments[ARG_CERT_PUB_DUMP].getArguments().firstElement(), this::writeRootFullCaCertificateToFile);`